O/Osi/build_tarballs.jl

```diff
@@ -25,6 +25,10 @@ export CXXFLAGS="-std=c++11"
 
 make -j${nproc}
 make install
+
+if [[ ${target} == "*mingw* ]]; then
+    cd $prefix/bin && mv libOsi-1.dll libOsi.dll && mv libOsiCommonTests-1.dll libOsiCommonTests.dll
```

Comment:
In this case we use to tweak the name of the product files instead of moving files :wink:

Reply:
I accidentally hit the merge button.

If we tweak the product files, then I need to account for both names when I build Clp. This just seemed easier.